### PR TITLE
Best practice database url

### PR DESCRIPTION
**Pull request by alonisser**
Moving to best practice database connection uri handling: using a standard uri parser and moving to an (optional) environment variable to supply DATABASE_URL(see http://12factor.net/config)

### CODE REVIEW

**Comment by alonisser** [comment]
Also fixing the missing logs folder with an empty .keep folder , to remove newcomers friction

File: kikar_hamedina/kikar_hamedina/settings.py [hunk 1]
```diff
@@ -123,15 +125,18 @@
...
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.postgresql_psycopg2',
+#         'NAME': 'kikar',
+#         'USER': 'kikar',
+#         'PASSWORD': 'kikar',
+#         'HOST': 'localhost'
+#     }
+# }
+config = dj_database_url.config(default='postgres://kikar:kikar@localhost:5432/kikar')
```
Comment (yotammanor) [inline kikar_hamedina/kikar_hamedina/settings.py]: @alonisser - in this config line, which part is db_name,user,password?

**Comment by yotammanor** [comment]
@alonisser Another thing to take into consideration - the DATABASES definition is usually defined in (ignored file) local_settings.py. Therfore, this change should be reflected in local_settings.py.template as well

File: kikar_hamedina/kikar_hamedina/settings.py (hunk 1 above)
Comment (alonisser) [inline kikar_hamedina/kikar_hamedina/settings.py]: @yotammanor see [here](https://github.com/kennethreitz/dj-database-url) :
urischeme://username:password@uri_to_db:port/db_name

This is a URI standard. and thus a convention used in the almost all services: rabbitmq, mongodb, redis, elasticsearch. etc
web URL which is a specific type of URI also conforms:
for example for a site with basic http auth you can use
http(schema type)://user:password@url_to_website/specific_resource

**Comment by alonisser** [comment]
@yotammanor actually now I checked it, the template does not handle DATABASES. just the settings file.. so no need to change there

**Pull request merged**